Source/Objects.lua

```diff
@@ -17,7 +18,8 @@ Auctionator = {
     CurrentPane = {
       UINeedsUpdate = false
     },
-    LiveDB = nil
+    LiveDB = nil,
+    InitialScanComplete = false
```

Comment:
Lets make this
```
  LiveDB = nil,
  Scans = {
    InitialScanComplete = false
  },
  -- etc
```
I'm going to put a bunch of stuff into `Scans` eventually; nice to have it all grouped together.